src/cryptography/hazmat/primitives/kdf/argon2.py

```diff
@@ -0,0 +1,11 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the BSD License. See the LICENSE file in the root of this repository
+# for complete details.
+
+from __future__ import annotations
+
+from cryptography.hazmat.bindings._rust import openssl as rust_openssl
+from cryptography.hazmat.primitives.kdf import KeyDerivationFunction
+
+Argon2id = rust_openssl.kdf.Argon2id
```

Comment:
Is this the righ capitalization?

Reply:
Yes, RFC 9106 also uses "Argon2id" in its capitalization.